pkg/operator/ceph/controller/spec.go

```diff
@@ -36,6 +36,7 @@ import (
 const (
 	// ConfigInitContainerName is the name which is given to the config initialization container
 	// in all Ceph pods.
+	// DefaultServiceAccount is a  service-account used for components that do not specify a dedicated service-account.
```

Comment:
remove this

Reply:
How about moving this comment down to the line immediately before the declaration of `DefaultServiceAccount`?